single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -30,6 +30,8 @@ This sample focuses on constructing a feature engineering system using Azure ML
     - [Data lineage](#data-lineage)
     - [Feature lineage](#feature-lineage)
   - [Verify the features in Feature Store](#verify-the-features-in-feature-store)
+  - [Exploratory data analysis (EDA)](#exploratory-data-analysis-eda)
+  - [Data validation](#data-validation)
 - [Model training and inferencing](#model-training-and-inferencing)
   - [Model training](#model-training)
   - [Model inferencing](#model-inferencing)
```

Comment:
Update line 53:

In addition to the main flow, there are optional steps for performing 'exploratory data analysis' and 'data validations' (illustrated by dotted lines in the diagram). These features are not part of the main processing steps, but are covered separately under [Additional Features](#additional-features).

Reply:
OH! Forgot to update this part accordingly, thanks for the reminder!